docs/kubernetes.md

```diff
@@ -0,0 +1,104 @@
+# Using Kubernetes Flex Volume Plugin
+
+As a Kubernetes flex volume plugin, k8s driver binary works with Convoy daemon to provide volume management tasks for uses using Kubenetes.
+
+## Register Flex Volume plugin to Kubernetes
+
+Please make sure Kubernetes v1.3.4+ are installed. Create a folder(vendor~driver) on the Kubelet volume plugin path: /usr/libexec/kubernetes/kubelet-plugins/volume/exec
+```
+sudo mkdir -p /usr/libexec/kubernetes/kubelet-plugins/volume/exec/rancher.io~k8s
```

Comment:
I think it's probably better as `rancher~convoy-k8s` instead of `rancher.io~k8s`.